spyderlib/plugins/shortcuts.py

```diff
@@ -176,6 +138,7 @@ def __init__(self, parent, context, name, sequence, shortcuts):
         self.text_new_sequence = CustomLineEdit(self)
         self.text_new_sequence.setPlaceholderText(sequence)
         self.helper_button = HelperToolButton()
+        self.label_warning = QLabel('<br><br>')

```

Comment:
I think this widget should be initialized just as `QLabel()` and hidden by default, unless there's a warning to show.

I say it because right now the dialog looks like this

![seleccion_003](https://cloud.githubusercontent.com/assets/365293/8635259/50aed45a-27e1-11e5-92d7-d0c06bc8f617.png)

which (to me) it's a bit ugly because there's too much space between the line edit and the buttons below it